README.md

```diff
@@ -252,7 +252,7 @@ To work with Azure AI, you need an [Azure OpenAI resource](https://learn.microso
 
 * AZURE_API_VERSION - e.g. 2024-02-15-preview
 * AZURE_API_BASE - e.g. https://my-org.openai.azure.com/
-* AZURE_OPENAI_API_KEY (optional) - equivalent to the `--api-key` cli argument
+* AZURE_API_KEY (optional) - equivalent to the `--api-key` cli argument
```

Comment:
in main.py I would update the help since it still says:

```
opt_api_key: Optional[str] = typer.Option(
    None,
    help="API key to use for the LLM (if not given, uses environment variables OPENAI_API_KEY or AZURE_OPENAI_API_KEY)",
)
```

Reply:
yes, good point